cpp/CMakeLists.txt

```diff
@@ -77,7 +77,8 @@ if(CMAKE_BUILD_TYPE STREQUAL "Debug" OR CMAKE_BUILD_TYPE STREQUAL "DEBUG")
         setup_target_for_coverage_lcov(
             NAME coverage
             EXECUTABLE memilio-test
-            EXCLUDE "${CMAKE_SOURCE_DIR}/tests*" "${CMAKE_SOURCE_DIR}/simulations*" "${CMAKE_SOURCE_DIR}/examples*" "${CMAKE_BINARY_DIR}/*" "/usr*"
+            EXCLUDE "${CMAKE_SOURCE_DIR}/tests*" "${CMAKE_SOURCE_DIR}/simulations*" "${CMAKE_SOURCE_DIR}/examples*"
+            "${CMAKE_BINARY_DIR}/*" "/usr*" "*.F"

```

Comment:
maybe add where the fortran file lies or is this not possible?

Reply:
I excluded *.F-files from the test coverage in a new commit. Now the test coverage on the CI machines runs without failure.